pages/common/nasm.md

```diff
@@ -0,0 +1,23 @@
+# nasm
+
+> The Netwide Assembler, a portable 80x86 assembler.
+
+- Assemble raw binary file `source` from `source.asm`:
+
+`nasm {{source.asm}}`
+
+- Assemble and generate an assembly listing file:
```

Comment:
Assemble an assembly file :) i.e. a program written in assembly language. And generate a listing file from it.

@gttotev - please correct me if I am wrong.